.github/workflows/releases.yaml

```diff
@@ -0,0 +1,51 @@
+name: Releases
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  PUSH: 'true'
+  DIRECT: 'true'
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  releases:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      - name: Set up QEMU
```

Comment:
QEMU is for cross-compile correct?

Reply:
Is for building multi-architecture images.

Reply:
https://github.com/docker/setup-qemu-action#about